src/server/__mockData__/mockData.js

```diff
@@ -81,7 +81,7 @@ export const mockData =  {
       logoUrl: "https://www.adewale.com"
     },
     parties: {
```

Comment:
the mock data contains only one party, I makes sense because you are also using it in making a post request in you test file